src/main/java/com/elo7/probe_spring/models/Probe.java

```diff
@@ -1,84 +1,82 @@
 package com.elo7.probe_spring.models;
 
-import com.elo7.probe_spring.exceptions.ProbeCollisionException;
-import com.elo7.probe_spring.exceptions.ProbeOutOfPlateauException;
-
 import javax.persistence.*;
+import javax.validation.constraints.NotNull;
 
 @Entity
 @Table(name = "probe")
 public class Probe {
```

Comment:
Colocar @annotation para que criar as constantes, not null e reference de chave estrageira.